config/initializers/sidekiq.rb

```diff
@@ -1,24 +1,6 @@
 require "govuk_sidekiq/sidekiq_initializer"
 
-if ENV["RACK_ENV"] == "test"
-  redis_config = {
-    host: ENV.fetch("REDIS_HOST", "127.0.0.1"),
-    port: ENV.fetch("REDIS_PORT", 6379),
-    namespace: "search-api-test",
-  }
-
-  Sidekiq.configure_server do |config|
-    config.redis = redis_config
-  end
-
-  Sidekiq.configure_client do |config|
-    config.redis = redis_config
-  end
-else
-  redis_config = {
-    host: ENV.fetch("REDIS_HOST", "127.0.0.1"),
-    port: ENV.fetch("REDIS_PORT", 6379),
-  }
-
-  GovukSidekiq::SidekiqInitializer.setup_sidekiq("search-api", redis_config)
-end
+GovukSidekiq::SidekiqInitializer.setup_sidekiq(
+  ENV.fetch("GOVUK_APP_NAME", "search-api"),
+  {},
```

Comment:
Feels like `{}` might be quite a common value here now, and without knowing the `govuk_sidekiq` library it's very unclear what `{}` is referring to.

Before changing lots of apps, could we change `govuk_sidekiq` to be:

```rb
def setup_sidekiq(govuk_app_name, redis_config: {})
```

Reply:
That's a good point. I think it's only this app that is affected by this - as all the others use the Railtie. In a way this app is using the backdoor.

I have to make a breaking change for govuk_sidekiq to kill the REDIS_HOST and REDIS_PORT env vars so I can include something to make this cleaner as part of that.

Reply:
Makes sense 👍🏻